src/components/Footer/styles.css

```diff
@@ -0,0 +1,521 @@
+@import url('https://rsms.me/inter/inter.css');
+html { font-family: 'Inter', sans-serif; }
+@supports (font-variation-settings: normal) {
+  html { font-family: 'Inter var', sans-serif; 
+    background: #F2F2F2;}
+}
+
+footer {
+    background: #FFFFFF;
+}
+
+#footer-component {
+    width: 100%;
+    display: grid;
+    justify-content: center;
+    text-align: center;
+    background: #FFFFFF;
+    margin: 0px auto 0px auto;
+    bottom:0; 
+}
+
+#become-nanny {
+    width: 473.8px;
+    height: 24px;
+    margin: 0 0 20px;
+    font-family: Inter;
+    font-size: 28px;
```

Comment:
You're using pixels, we use rem for basically everything.

Here's one article that explains a bit why, although there are some more things.

Also, keep in mind that on some projects we have different patterns, but usually we use just rem.

https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9